src/blog/first-100k-lines.vmd

```diff
@@ -0,0 +1,415 @@
+---
+title: How to Survive Your First 100,000 Lines
+author: Evan Ovadia
+date: Apr 9, 2022
+realm: blog
+path: blog/first-100k-lines
+layout: annotated
+namespace: c-blog m-annotated
+sponsor: us
+---
+
+
+
+The Vale compiler just hit its [100,000th line of code.](linkhere)
```

Comment:
Add timeframe, "after X years of development the compiler hit its ..." to give readers a frame of reference.